tests/.e2e-env

```diff
@@ -1,12 +1,9 @@
 POSTGRES_USER="postgres"
 POSTGRES_PASSWORD="postgres"
-POSTGRES_DB="data"
-POSTGRES_TABLE="medic"
+POSTGRES_DB="cht_sync"
+POSTGRES_TABLE="couchdb"
```

Comment:
yay!  makes so much more sense (and I see keeps these in sync with [default values](https://github.com/medic/cht-sync/blob/5bfbccecd0735e009f98af092552a5208305c62a/env.template#L4C1-L6C24)).

Reply:
Yes, it should have been updated when the DBT environment variables were removed.